translate/storage/cpo.py

```diff
@@ -492,8 +502,9 @@ def _domarkfuzzy(self, present=True):
         gpo.po_message_set_fuzzy(self._gpo_message, present)
 
     def makeobsolete(self):
-        # FIXME: libgettexpo currently does not reset other data, we probably want to do that
-        # but a better solution would be for libgettextpo to output correct data on serialisation
+        # FIXME: libgettexpo currently does not reset other data, we probably
+        # want to do that but a better solution would be for libgettextpo to
+        # output correct data on serialisation
```

Comment:
As per our styleguide http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#comments Write sentences correctly: proper capitalisation and punctuation.